cmds/report.c

```diff
@@ -230,16 +264,27 @@ static void report_functions(struct uftrace_data *handle, struct opts *opts)
 	if (uftrace_done)
 		return;
 
-	if (avg_mode == AVG_NONE)
-		pr_out(f_format, "Total time", "Self time", "Calls", maxlen, "Function");
-	else if (avg_mode == AVG_TOTAL)
-		pr_out(f_format, "Avg total", "Min total", "Max total", maxlen, "Function");
-	else if (avg_mode == AVG_SELF)
-		pr_out(f_format, "Avg self", "Min self", "Max self", maxlen, "Function");
-
-	pr_out(f_format, line, line, line, maxlen, line);
-
-	print_and_delete(&sort_root, true, NULL, print_function);
+	if (opts->format) {
```

Comment:
I don't think we have to assume `--format` will have only `csv`, so you should check it as follows:
```c
if (!strcmp(opts->format, "csv")) {
    ...
}
```

Reply:
Fixed